File: source/qa/python/qa/scalability/experiment/partition_shape/configuration.py
from lue.qa.scalability.core import json_to_data

from .. import configuration
from . import experiment


class Configuration(configuration.Configuration):
    def __init__(self, data):

        super(Configuration, self).__init__(data)
        self.from_json(data)

    def from_json(self, data):

        experiment_data = json_to_data(data["experiment"])
        experiment_data["command_pathname"] = self.command_pathname
        self.experiment = experiment.Experiment(experiment_data)

File: source/view/view/src/configuration.cpp
#include "lue/view/configuration.hpp"


namespace lue {
    namespace view {

        Configuration::Configuration():

            _show_details{false}

        {
        }


        bool Configuration::show_details() const
        {
            return _show_details;
        }


        bool& Configuration::show_details()
        {
            return _show_details;
        }

    }  // namespace view
}  // namespace lue
